api/api.proto

```diff
@@ -1585,3 +1585,81 @@ message SdkCloudBackupStateChangeRequest {
 
 message SdkCloudBackupStateChangeResponse {
 }
+
+enum CloudMigrateOperationType {
+    // Migrate all volumes in the cluster
+    MigrateCluster = 0;
```

Comment:
Highly suggest that 0 is left as an unknown value on all enums. That way you can check for errors if it was not defined.

Reply:
Added invalid states